src/python/issue_management/issue_tracker.py

```diff
@@ -50,6 +111,11 @@ def merged_into(self):
   def merged_into(self, new_merged_into):
     raise NotImplementedError
 
+  @property
+  def open(self):
```

Comment:
I think `is_open` would be less confusing.

Reply:
Done.